### PR TITLE
(SUP-1574) transfer from npwalker to puppetlabs

### PR DESCRIPTION
**Pull request by tkishel**
To be Release 1.0.0

### CODE REVIEW

**Comment by jarretlavallee** [comment]
This repo should be named `puppetlabs/puppetlabs-pe_databases` to comply with the naming standard of the Puppet modules.

**Comment by tkishel** [comment]
Gah

File: metadata.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  "author": "npwalker",
+  "name": "puppetlabs-pe_databases",
+  "version": "1.0.0",
+  "author": "puppetlabs",
   "summary": "A Puppet Module for Backing Up / Maintaining / Tuning Your Puppet Enterprise Databases",
   "license": "Apache-2.0",
-  "source": "https://github.com/npwalker/pe_databases",
-  "project_page": "https://github.com/npwalker/pe_databases",
-  "issues_url": "https://github.com/npwalker/pe_databases/issues",
+  "source": "https://github.com/puppetlabs/pe_databases",
```
Comment (jarretlavallee) [inline metadata.json]: This should be good to go after the updated URL here.

**Comment by jarretlavallee** [comment]
@asselvakumar The repo URL needs to be updated in several places if you want to push a commit to fix it. It should be <https://github.com/puppetlabs/puppetlabs-pe_databases>

**Comment by tkishel** [comment]
Done!

**Comment by m0dular** [comment]
Merging but holding off on cutting a release until we fix the tests.

**Pull request merged**